src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -43,18 +42,26 @@ const formatItem = (item, env) => {
   return assocPath(['locators', 'assetUri'], `${baseUri}${uriSuffix}`, item);
 };
 
-const CpsRelatedContent = ({ content }) => {
+const CpsRelatedContent = ({ content, noGhostGrid }) => {
   const { script, service, dir, translations } = useContext(ServiceContext);
   const { env } = useContext(RequestContext);
+  const ConditionalWrapper = ({ condition, wrapper, children }) =>
+    condition ? wrapper(children) : children;
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'related-content-heading',
+  };
   if (!content.length) return null;
 
   return (
-    <GhostGrid
-      as="section"
-      role="region"
-      aria-labelledby="related-content-heading"
+    <ConditionalWrapper
+      condition={!noGhostGrid}
```

Comment:
@HarveyPeachey is it possible to make this work. without the extra negation on `noGhostGrid` ? 
perhaps altering this bit 
```
condition ? children : wrapper(children);
```

Reply:
Yep that makes sense